docs/usage/providers.md

```
@@ -312,6 +312,165 @@ echo "$response"
 
 ### Enterprise Providers
 
+<details>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: details

(MD033, no-inline-html)